OpenRA.Platforms.Default/Sdl2Input.cs

```diff
@@ -161,7 +161,11 @@ public void PumpInput(IInputHandler inputHandler)
 								Platform.CurrentPlatform == PlatformType.Windows)
 								Game.Exit();
 							else
-								inputHandler.OnKeyInput(keyEvent);
+							{
+								// Only register initial key press
+								if (e.key.repeat == 0)
```

Comment:
I'm following the initial implementation, specifically the Special case workaround for windows users If this is not needed then the normal programme flow of a key press whatever that is, is  followed. - I would argue an `else` is a better use here to show that.

It would also allow in future if more of these special cases where needed they could be added as an `else if` while the normal flow is the else.

That was my thinking behind it anyway.